### PR TITLE
Update Scenarios Intro.md

### PR DESCRIPTION
**Pull request by rowa97**

### CODE REVIEW

File: articles/testing/overview.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
 
+Following the rise of APIs, it has become increasingly important to develop highly flexible, performant, and powerful testing roadmaps to catch bugs faster and rapidly iterate without breaking existing features. A thorough test suite is essential for: 
+- Assessing the health of an API
+- Providing valuable documentation 
+- Driving design and implementation 
+- Managing technical debt 
+
+## Assessing the Health of an API
+
+APIs require maintenance like any other software. To ensure that your API is functioning properly, a suite of test should be run periodically to check for weaknesses and errors.
```
Comment (rossmcdonald) [inline articles/testing/overview.md]: a suite of test -> a suite of tests

File: articles/testing/overview.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+- Assessing the health of an API
+- Providing valuable documentation 
+- Driving design and implementation 
+- Managing technical debt 
+
+## Assessing the Health of an API
+
+APIs require maintenance like any other software. To ensure that your API is functioning properly, a suite of test should be run periodically to check for weaknesses and errors.  
+
+>Create and [run tests within Stoplight](/testing/running-tests/in-stoplight), [the terminal](/testing/running-tests/in-the-terminal), or [trigger it by URL](/testing/running-tests/triggering-by-url). If you prefer to [use a CI](/testing/continuous-integration/overview), Stoplight also provides integrations to [Jenkins](/testing/continuous-integration/jenkins), [CircleCI](/testing/continuous-integration/circle-ci), and [Travis](/testing/continuous-integration/travis).
```
Comment (rossmcdonald) [inline articles/testing/overview.md]: use a CI -> use CI

File: articles/testing/overview.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+>Create and [run tests within Stoplight](/testing/running-tests/in-stoplight), [the terminal](/testing/running-tests/in-the-terminal), or [trigger it by URL](/testing/running-tests/triggering-by-url). If you prefer to [use a CI](/testing/continuous-integration/overview), Stoplight also provides integrations to [Jenkins](/testing/continuous-integration/jenkins), [CircleCI](/testing/continuous-integration/circle-ci), and [Travis](/testing/continuous-integration/travis).  
+
+## Providing Valuable Documentation & Driving Design
+
+API tests provide insight into how your API behaves under certain scenarios and can drive design if created early enough in the design process. Test/Behavior-driven development (TDD/BDD) encourages you to think about design requirements before writing any code. 
+
+>Stoplight further promotes this design first principle by providing [Contract Testing](testing/leveraging-openapi/contract-testing); an integration between your tests and your OpenAPI Specification. This allows for immediate validation and verification that your API responses match the “Contract” specified in your OpenAPI spec.
```
Comment (rossmcdonald) [inline articles/testing/overview.md]: The link needs a leading forward-slash.

File: articles/testing/overview.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+
+## Managing Technical Debt 
+
+Microservices and serverless architecture have made it easier than ever to iterate quickly. The downside is that it has become easier to introduce bugs and technical debt becomes unmanageable without the proper testing solution. It is critical to have a comprehensive test suite to allow teams to test the API during development.
```
Comment (rossmcdonald) [inline articles/testing/overview.md]: architecture -> architectures

Second sentence is hard to parse. Consider changing to:

> The downside of rapid development is an increase in bugs and technical debt, making projects harder to manage without a proper testing solution.

**Comment by rowa97** [comment]
I agree, will need to be revised when we have more time, thanks for the comments

File: articles/testing/overview.md (hunk 4 above)
Comment (rowa97) [reply articles/testing/overview.md]: well put!
